state/ArcgisStore/ArcgisStoreProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import 'core-js/stable/atob'
```

Comment:
might be worth it leaving a comment as to what this relates to

Reply:
the import is redundant there... will remove